org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
 {
-    "id": "org.musicbrainz.Picard",
+    "app-id": "org.musicbrainz.Picard",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
     "sdk": "org.freedesktop.Sdk",
     "command": "picard",
     "rename-desktop-file": "picard.desktop",
     "rename-icon": "picard",
     "finish-args": [
+        /* X11 + XShm access */
         "--share=ipc",
         "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL access */
+        "--device=dri",
```

Comment:
Is OpenGL really needed? I don't really see why Picard would use it. :confused:

Reply:
It's used by Qt. Without it, Picard actually launches but we got some OpenGL related warnings from Qt in the CLI.

Reply:
Ah, alright, thanks.